src/fs/class.node.test.ts

```diff
@@ -1016,16 +949,13 @@ describe("File System Class", async () => {
       await t.write(Path.file("public", "file"), "bytes", await t.read(Path.file("private", "file"), "bytes"))
     })
 
-    expect(
-      await fs.read(Path.file("public", "file"), "utf8")
-    ).toEqual(
+    assert.equal(
+      await fs.read(Path.file("public", "file"), "utf8"),
       "💃"
     )
   })
 
   it("doesn't commit a transaction when an error occurs inside of the transaction", async () => {
-    expect.assertions(1)
```

Comment:
Wasn't able to find an equivalent in `assert`, but the test passes so probably alright?

Reply:
It only fails in certain scenarios when the code doesn't properly work.
This is saying that the test should run exactly one assertion, no less.
I think we should replace this with something like this from the `assert` lib: https://nodejs.org/api/assert.html#trackerverify

Reply:
Oh yeah, how did I miss that. 😆 

Thanks for finding it. Updated the test case to use that.